tests/remote/remote_target.pm

```diff
@@ -25,11 +24,13 @@ sub run {
     assert_screen("remote_slave_ready", 350);
     mutex_create("installation_ready");
     # wait while whole installation process finishes
-    $self->wait_boot(bootloader_time => 1800);
+    mutex_wait("installation_done");
+    $self->wait_boot(bootloader_time => 120);
 }
 
 sub test_flags {
```

Comment:
Small comment, with `y2_installbase` we actually don't need this override, all `y2_installbase` are fatal.

Reply:
good catch! right, removed.